corehq/apps/hqwebapp/views.py

```diff
@@ -351,6 +351,11 @@ def login(req):
     # this view, and the one below, is overridden because
     # we need to set the base template to use somewhere
     # somewhere that the login page can access it.
+
+    if settings.SERVER_ENVIRONMENT == 'icds':
```

Comment:
it might be slightly less gross to have a boolean variable in ```settings``` that gets used here, but that might be unnecessarily complicated.